pocket_lab/pl_gui_iio_wrapper.c

```diff
@@ -363,7 +363,7 @@ int32_t pl_gui_read_chn_attr(char *attr_name, char *attr_val,
 			     uint32_t chn_indx, uint32_t dev_indx)
 {
 	struct iio_device *iio_dev;
-	struct iio_ch_info chn_info;
+	struct iio_ch_info chn_info = { 0 };
```

Comment:
Is channel info is not applicable here also?

Reply:
chn_info.ch_num = chn_indx;
This sets the channel number later in the function. I only need channel number for attributes read/write